openedx/features/course_experience/__init__.py

```diff
@@ -58,3 +59,15 @@ def course_home_url_name(course_key):
         return 'openedx.course_experience.course_home'
     else:
         return 'info'
+
+
+class CourseHomeMessages(UserMessageCollection):
```

Comment:
@HarryRein Thanks for the explanation. I'm fine with leaving it as is for now, and revisiting later if need be.